services/static-webserver/client/source/class/osparc/file/FilePicker.js

```diff
@@ -658,7 +658,9 @@ qx.Class.define("osparc.file.FilePicker", {
               this.__uploadedParts[chunkIdx]["e_tag"] = eTag.slice(1, -1);
               const uploadedParts = this.__uploadedParts.filter(uploadedPart => uploadedPart["e_tag"] !== null).length;
               const progress = uploadedParts/this.__uploadedParts.length;
-              this.getNode().getStatus().setProgress(100*progress-1);
```

Comment:
Q: When you describe that you get displayed negative numbers, isn't it what happens when the progress is actually 0? 
i.e. ``progress=0 -> 100*0-1 = -1`` and when the upload finished ``progress=1 -> 100*1-1 = 99``

wouldn't it be solved using a linear function as `` 98*progress +1`` ??

Reply:
Correct, that's yet another solution.